### ISSUE TITLE
Change the way mobile notifications are described

### ISSUE THREAD

**Issue by nithindavid** [chatwoot/chatwoot-mobile-app#255]
**Current**
- A new conversation ID-34 has been created

**Proposed**
- [New ticket] - Conv-34 has been created in chatwoot
- [Assigned to me] - Conv-34 has been assigned to you
- [New message] - New message in conv -34

**Why**
It's difficult to filter the idea from the notification message as every notif text looks the same.

**Comment by nithindavid** [comment]
@muhsin-k @pranavrajs @sojan-official What do you guys think ?

**Comment by pranavrajs** [comment]
@nithindavid This is a nice suggestion. I would like to make small changes to the text you proposed.


 - [New conversation] - #34 has been created in chatwoot
 - [Assigned to me] - #34 has been assigned to you
 - [New message - #34] - {{use the content of the message here}}

I think this should be a ticket in Chatwoot repo instead of the mobile app. You can transfer the ticket once @muhsin-k confirms.

**Comment by sojan-official** [comment]
yep the change has to happen in chatwoot repo @pranavrajs @nithindavid
